sdk/apimanagement/apimanagement-custom-widget-scaffolder/review/apimanagement-custom-widget-scaffolder.api.md

```diff
@@ -0,0 +1,38 @@
+## API Report File for "@azure/apimanagement-custom-widget-scaffolder"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export function generateProject(widgetConfig: TCustomWidgetBaseConfig, deployConfig: TDeployConfig, miscConfig?: TMiscConfig): Promise<void>;
+
+// @public
+export const OVERRIDE_DEFAULT_PORT = 3000;
+
+// @public
+export const OVERRIDE_PORT_KEY = "MS_APIM_CW_localhost_port";
+
+// @public
+export interface TCustomWidgetBaseConfig {
+    displayName: string;
+    iconUrl?: string;
+    tech: TScaffoldTech;
```

Comment:
Well neither TS nor React are frameworks so I don't think it makes sense to name it "framework".
It will have 3 options: TS - language, React - library, Vue - framework. Therefor "tech" was IMO best name I came up with.